lib/src/components/NxTable/types.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import {TableHTMLAttributes} from 'react';
+
+// Final Props are the HTMLProps & our re-definitions
```

Comment:
Comment copied from another file?  There aren't any redefinitions here

Reply:
It sure is. :P

Reply:
removed